examples/mint-repl/sample_runs/reissue_ez.txt

```diff
@@ -27,62 +27,59 @@ Number of Mint Nodes: 1
+e13050c7dd443e07e515eb2b901460da291447eb3bd6773c5289d369fb7f36c6b3b932f5340a290bc0f3d5c248adad840000000000000001cc95d3e6df19fe4e971c28cb59150f63805a8ec878011fd289c80593b6bd5906a84f1360cf1fb4a7e6a92d1d95b26e0ffff233c6f0eaa196d94cfe5ee4af705627eae334ffb2ddc764f37071e271afe2a8efb85886a3e0a06d463fa0b4551581e13050c7dd443e07e515eb2b901460da291447eb3bd6773c5289d369fb7f36c6b3b932f5340a290bc0f3d5c248adad84bbc622db0af03afbef1a7af93fe8556c58ac1b173f3a4ea105b974974f8c68c30faca94f8c63952694d79731a7d3f1970000000000000001e13050c7dd443e07e515eb2b901460da291447eb3bd6773c5289d369fb7f36c6b3b932f5340a290bc0f3d5c248adad840000000000000001bbc622db0af03afbef1a7af93fe8556c58ac1b173f3a4ea105b974974f8c68c30faca94f8c63952694d79731a7d3f1970000000000000001e13050c7dd443e07e515eb2b901460da291447eb3bd6773c5289d369fb7f36c6b3b932f5340a290bc0f3d5c248adad84017c747ab2424f87bce27c850c361fb2c8e24b853d8345819685f3f7c0704d820e6350eada09d75fe44f28ad0f4531136695b912bb1cab320b14b27d4eaf32492c802034e1eb2bd33f70ae92e08757f8ac890bd9f53b019c8ffa892d84a4b2747cd1683bd8afd02449ab9276f8137f7bae23eedef41aa579c65f146b3693a1da343a4eb4c06401ba28c664768494d76e3eed8b6ae466a7bb7a7286f8cc15a85046084364f7029736f18121fed3006a8cdd933f11d60c10d81d952d9b880e69006abcab77d787af61194d6971fac1250b002241a065b48380a5cf86b2f881744a59d244b2e0f41060ea3917f944abe115e305408b79ff8e501f5509461b092f960048c396ff9f2f5a92ccbd68022e625404c9c603b1c3a7b19b6c906be4ea0c1a680ca39bac19dc38cc6ee811586459aee280c9a48f87e2a48acecc2cfbdb851c72eb96612e2ef206705bd4d36f2b27a94f36a51e10b82743854de0bbf80cf95874d9ae917dc588337b3d1d4c91497caac22cc836534597cc58cef6743ed4d5724756d1b22d95c7ab4b8d11c0785f60b34bce91afb47f773aba0fa4c67bbd0cde38baaf761ed2d301b83ac8a07c9641fe7c3745e96e8f273bb0be2c8ff458e2440658fcb14c3e20768df414712535e5dc392012e58aee62f4721712a9b0e6692b02ab7e1f8e5c80a158577d332ecb8190ec20aaa401d6da42286c4d7eb7fc38c60f059ed05f1c705066bbe02f0486c686362c429d99e534e7dcc341a789c70cae28152dacf0b24271b4b4cc90b8c0ace262c09725b4fd626d3a5158f7df8fd3ee6162a65c700051851b125403f416029b955e5bc770b6f7d119f7273460bd70a24169d7db70055430481dbcf2cf9c1c6a390df44be21fa58cde20b48b135c215845fc590105babb7f60668753de793d3bed1c12abcccb5b218022357aeba4785a00000000000002a04d8ea7ca1e65de3f6c5bcc481c6fbe78e77b28f6fc255604d18b4c7f9b3b05bc32ce433590953a49e0d7fb64ab3f592ac41d25a5802ef0bbf517a1e384652c002002015b7fdc89f960743eb645076c0c23845e3135bd18e47dee60c23ef647917d5e277ba4a0c002dd80436aa7ebc80fb55864967a2f37a1fdb6fbae737f53921ad8d7da966423d88d68c34388fa265b1470ae1fe9a2d69767eb193b3ff6074d9d638bc210f25b1700000000000000287634ac86d6ee1afbac7b3ea56c6b281945562b035f958efdbc645f8213fe9a49df0c54d127c767ecbbe7f0dcfa415ab7bbc622db0af03afbef1a7af93fe8556c58ac1b173f3a4ea105b974974f8c68c30faca94f8c63952694d79731a7d3f1970000000000000001
 
 
 -- SpendBook --
 
-  0000000000000000000000000000000000000000000000000000000000000000
-
+  97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb
```

Comment:
Would it make any sense to hash the key so it's smaller, eg as spendbook key?   Or would that break key derivation somehow?

Reply:
Yeah, I think we can safely do that, but I'd like to keep things unobscured, at least the first pass